### PR TITLE
cli: check for args in explain command

### PR DESCRIPTION
**Pull request by bshore**
Aims to resolve https://github.com/tilt-dev/tilt/issues/6055

### CODE REVIEW

File: internal/cli/explain.go [hunk 1]
```diff
@@ -95,6 +96,10 @@ func (c *explainCmd) run(ctx context.Context, args []string) error {
 	a.Incr("cmd.explain", cmdTags.AsMap())
 	defer a.Flush(time.Second)
 
+	if len(args) == 0 {
```
Comment (nicks) [inline internal/cli/explain.go:99]: this is a good fix! but we usually try to keep this consistent with what upstream kubectl is doing. in this case, i think it's missing a call to Validate() after the Complete() call a few lines below this. does that work?
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/explain/explain.go#L141

Comment (bshore) [reply internal/cli/explain.go:99]: Yeah that's much better, thanks!

**Review by nicks — APPROVED** [review]
LGTM!

aw, i wish you had kept the test!

**Comment by bshore** [comment]
> LGTM!
> 
> aw, i wish you had kept the test!

@nicks I wish I could have, too! Unfortunately the upstream `Validate()` function calls `os.Exit()` so there was no easy way to assert/require on any output 😞

**Pull request merged**